### DISCUSSION TITLE
WPF source code missing in Source Browser

### DISCUSSION THREAD

**Discussion by batzen** [dotnet/wpf#6410] [General]
Since a few days or weeks, i don't know exactly since when, WPF source code is missing on https://source.dot.net
As i don't know which team is responsible and since WinForms is still being included there i thought this might be the correct repo to report this issue.
If this is the wrong repo it would be nice to get a hint at where this should be reported or if it gets re-routed to the correct repo.

For example, browsing to https://source.dot.net/#q=frameworkelement only lists source code from PresentationBuildTasks but not from WPF itself.

**Comment by ThomasGoulet73** [comment]
@batzen Maybe a better repo would be https://github.com/KirillOsenkov/SourceBrowser ?

**Comment by batzen** [comment]
That's just the repo containing the sources for Source Browser itself. Maybe @KirillOsenkov can point to the right repo or team?

**Reply by KirillOsenkov** [reply]
@alexperovich

**Comment by alexperovich** [comment]
There are a bunch of build errors for wpf in the source-indexer build that look like this:
```
D:\a\_work\1\s\bin\repo\wpf\src\Microsoft.DotNet.Wpf\src\DirectWriteForwarder\CPP\TrueTypeSubsetter\util2.cpp(27): error C2253: 'System::IParsable::Parse': pure specifier or abstract override specifier only allowed on virtual function [D:\a\_work\1\s\bin\repo\wpf\src\Microsoft.DotNet.Wpf\src\DirectWriteForwarder\DirectWriteForwarder.vcxproj]
```

These are preventing the C# from building at all

**Reply by ThomasGoulet73** [reply]
@alexperovich This error is because your version of C++/CLI compiler does not support static abstract interface members. You need Visual Studio 2022 17.2 Preview 2 to build WPF .Net 7. See dotnet/docs#28542

**Comment by alexperovich** [comment]
This is now fixed.

**Comment by singhashish-wpf** [comment]
Thank you @alexperovich !
Thanks @batzen and  @ThomasGoulet73 too!
